### PR TITLE
Added Blazor samples

### PR DESCRIPTION
**Pull request by AkashArul26**
Blazor samples for below UG pages : 
https://help.syncfusion.com/file-formats/docio/working-with-word-document#working-with-word-document-properties
https://help.syncfusion.com/file-formats/docio/loading-and-saving-document#opening-an-existing-document

### CODE REVIEW

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]
Change the sample names properly

Open-and-save-Word-document
Modify-built-in-document-properties

File: Read-and-Save-document/Open-and-save-Word-document/Blazor/Client-side-application/Create-Word-document/Pages/DocIO.razor [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<button class="btn btn-primary" @onclick="@CreateWord">Create Word</button>
+
+@code {
+    @functions {
+
+        async void CreateWord()
+        {
+            using (Stream inputStream = await client.GetStreamAsync("sample-data/HelloWorld.docx"))
+            {
+                //Creating a new document.
```
Comment (Suriya-Balamurugan) [inline Read-and-Save-document/Open-and-save-Word-document/Blazor/Client-side-application/Create-Word-document/Pages/DocIO.razor:19]: Change this XML comments

File: Word-document/Modify-built-in-document-properties/Blazor/Client-side-application/Create-Word-document/Pages/DocIO.razor [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<button class="btn btn-primary" @onclick="@CreateWord">Create Word</button>
+
+@code {
+    @functions {
+
+        async void CreateWord()
+        {
+            using (Stream inputStream = await client.GetStreamAsync("sample-data/Template.docx"))
+            {
+                //Creating a new document.
```
Comment (Suriya-Balamurugan) [inline Word-document/Modify-built-in-document-properties/Blazor/Client-side-application/Create-Word-document/Pages/DocIO.razor:19]: Change this XML comments

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

**Comment by Suriya-Balamurugan** [comment]
Use proper commit message and title for the PR

File: Read-and-Save-document/Open-and-save-Word-document/Blazor/Server-side-application/Create-Word-document/Data/WordService.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+using System.IO;
+
+
+namespace Create_Word_document.Data
+{
+    public class WordService
+    {
+        public MemoryStream CreateWord()
+        {
+            FileStream sourceStreamPath = new FileStream(@"wwwroot/data/HelloWorld.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
```
Comment (Suriya-Balamurugan) [inline Read-and-Save-document/Open-and-save-Word-document/Blazor/Server-side-application/Create-Word-document/Data/WordService.cs:13]: Use using statement for file stream

File: Word-document/Modify-built-in-document-properties/Blazor/Server-side-application/Create-Word-document/Data/WordService.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+using System.IO;
+
+
+namespace Create_Word_document.Data
+{
+    public class WordService
+    {
+        public MemoryStream CreateWord()
+        {
+            FileStream sourceStreamPath = new FileStream(@"wwwroot/data/Template.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
```
Comment (Suriya-Balamurugan) [inline Word-document/Modify-built-in-document-properties/Blazor/Server-side-application/Create-Word-document/Data/WordService.cs:13]: Use using statement for file stream

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: Read-and-Save-document/Open-and-save-Word-document/Blazor/Client-side-application/Open-and-save-Word-document/Pages/DocIO.razor.bak [hunk 5]
```diff
@@ -0,0 +1,37 @@
+﻿@page "/DocIO"
```
Comment (Suriya-Balamurugan) [inline Read-and-Save-document/Open-and-save-Word-document/Blazor/Client-side-application/Open-and-save-Word-document/Pages/DocIO.razor.bak:1]: Remove this unwanted file DocIO.razor.bak

File: Word-document/Modify-built-in-document-properties/Blazor/Server-side-application/Modify-built-in-document-properties/Data/WordService.cs.bak [hunk 6]
```diff
@@ -0,0 +1,32 @@
+﻿using Syncfusion.DocIO;
```
Comment (Suriya-Balamurugan) [inline Word-document/Modify-built-in-document-properties/Blazor/Server-side-application/Modify-built-in-document-properties/Data/WordService.cs.bak:1]: Remove this unwanted file WordService.cs.bak

File: Read-and-Save-document/Open-and-save-Word-document/Blazor/Server-side-application/Open-and-save-Word-document/Data/WordService.cs.bak [hunk 7]
```diff
@@ -0,0 +1,28 @@
+﻿using Syncfusion.DocIO;
```
Comment (Suriya-Balamurugan) [inline Read-and-Save-document/Open-and-save-Word-document/Blazor/Server-side-application/Open-and-save-Word-document/Data/WordService.cs.bak:1]: Remove this unwanted file WordService.cs.bak

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: Read-and-Save-document/Open-and-save-Word-document/Blazor/Client-side-application/Create-Word-document/Pages/DocIO.razor (hunk 1 above)
Comment (AkashArul26) [reply Read-and-Save-document/Open-and-save-Word-document/Blazor/Client-side-application/Create-Word-document/Pages/DocIO.razor]: Addressed the requested changes

File: Word-document/Modify-built-in-document-properties/Blazor/Client-side-application/Create-Word-document/Pages/DocIO.razor (hunk 2 above)
Comment (AkashArul26) [reply Word-document/Modify-built-in-document-properties/Blazor/Client-side-application/Create-Word-document/Pages/DocIO.razor]: Addressed the requested changes

File: Read-and-Save-document/Open-and-save-Word-document/Blazor/Server-side-application/Create-Word-document/Data/WordService.cs (hunk 3 above)
Comment (AkashArul26) [reply Read-and-Save-document/Open-and-save-Word-document/Blazor/Server-side-application/Create-Word-document/Data/WordService.cs]: Addressed the requested changes

File: Word-document/Modify-built-in-document-properties/Blazor/Server-side-application/Create-Word-document/Data/WordService.cs (hunk 4 above)
Comment (AkashArul26) [reply Word-document/Modify-built-in-document-properties/Blazor/Server-side-application/Create-Word-document/Data/WordService.cs]: Addressed the requested changes

File: Read-and-Save-document/Open-and-save-Word-document/Blazor/Client-side-application/Open-and-save-Word-document/Pages/DocIO.razor.bak (hunk 5 above)
Comment (AkashArul26) [reply Read-and-Save-document/Open-and-save-Word-document/Blazor/Client-side-application/Open-and-save-Word-document/Pages/DocIO.razor.bak]: Addressed the requested changes

File: Word-document/Modify-built-in-document-properties/Blazor/Server-side-application/Modify-built-in-document-properties/Data/WordService.cs.bak (hunk 6 above)
Comment (AkashArul26) [reply Word-document/Modify-built-in-document-properties/Blazor/Server-side-application/Modify-built-in-document-properties/Data/WordService.cs.bak]: Addressed the requested changes

File: Read-and-Save-document/Open-and-save-Word-document/Blazor/Server-side-application/Open-and-save-Word-document/Data/WordService.cs.bak (hunk 7 above)
Comment (AkashArul26) [reply Read-and-Save-document/Open-and-save-Word-document/Blazor/Server-side-application/Open-and-save-Word-document/Data/WordService.cs.bak]: Addressed the requested changes

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**